packages/nextjs/app/builders/_components/MembersList.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+import { AiOutlineLoading3Quarters } from "react-icons/ai";
+import { RiShareBoxLine } from "react-icons/ri";
+import { Address } from "~~/components/scaffold-eth";
+import { useScaffoldEventHistory, useScaffoldReadContract } from "~~/hooks/scaffold-eth";
+
+const MembersList = ({ builders }: { builders: string[] }) => {
+  // Getting the list of builders who have completed their check-in.
+  const { data: checkedInnBuilders, isLoading: loading } = useScaffoldReadContract({
+    contractName: "BatchRegistry",
+    functionName: "checkedInCounter",
+  });
+
+  // Getting the list of builders' address list who have completed their check-in.
+  const { data: MembersList } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    blockData: true,
+    fromBlock: 123753708n - 123506774n,
+    transactionData: true,
+    receiptData: true,
+  });
+
+  // Filtering the list to count the number of builders with a profile page.
+  const profileBuildersCount = MembersList?.filter(member => {
+    const address = member?.args?.builder;
+    return address && builders.includes(address);
+  }).length;
+
+  return (
+    <>
+      <div className=" flex justify-center items-center md:gap-x-10 relative">
+        <div className="flex items-center gap-x-2 md:mb-8 mb-5">
+          <p className="text-lg font-normal text-[25px] underline">Buildes :</p>
```

Comment:
Typo: "Buildes :"

Reply:
Fixed the typo. "Buildes:" to "Builders:"